Auto-merge main -> dev (#127)
repowerednl/.github: README.md

- For using such a template in a repository: [Use template](https://docs.github.com/en/actions/writing-workflows/using-workflow-templates)
- For testing such a template including explanations: [workflow-tests](https://github.com/repowerednl/workflow-tests)

### GitHub App token

> Bot actions (pushing a commit, creating a pull request, merging it) are done with a token of our GitHub App instead of `secrets.GITHUB_TOKEN` or a (a big no-go!) Personal Access Token (PAT). That keeps the action traceable to the app, gives it the rights it needs on protected branches and - unlike `GITHUB_TOKEN` or PATs - the events it creates do trigger other workflows.

Reviewer: If we don't want to allow PATs, maybe only mention the GitHub App token.

```suggestion
Bot actions (pushing a commit, creating a pull request, merging it) are done with a token of our GitHub App instead of `secrets.GITHUB_TOKEN`. That keeps the action traceable to the app, gives it the rights it needs on protected branches and - unlike `GITHUB_TOKEN` - the events it creates do trigger other workflows. The use of personal access tokens (PATs) is not allowed.
```
